RATapi/controls.py

```diff
@@ -39,40 +40,98 @@
 class Controls(BaseModel, validate_assignment=True, extra="forbid"):
     """The full set of controls parameters for all five procedures that are required for the compiled RAT code."""
 
+    model_config = ConfigDict(use_attribute_docstrings=True)
+
     # All Procedures
     procedure: Procedures = Procedures.Calculate
+    """Which procedure RAT should execute. Can be 'calculate', 'simplex', 'de', 'ns', or 'dream'."""
+
     parallel: Parallel = Parallel.Single
+    """How the calculation should be parallelised. Can be 'single', 'contrasts' or 'points'."""
+
     calcSldDuringFit: bool = False
+    """Whether SLD will be calculated during fit (for live plotting etc.)"""
+
     resampleMinAngle: float = Field(0.9, le=1, gt=0)
+    """The upper threshold on the angle between three sampled points for resampling, in radians over pi."""
+
     resampleNPoints: int = Field(50, gt=0)
+    """The number of initial points to use for resampling."""
+
     display: Display = Display.Iter
+    """How much RAT should print to the terminal. Can be 'off', 'iter', 'notify', or 'final'."""
+
     # Simplex
     xTolerance: float = Field(1.0e-6, gt=0.0)
+    """[SIMPLEX] The termination tolerance for step size."""
+
     funcTolerance: float = Field(1.0e-6, gt=0.0)
+    """[SIMPLEX] The termination tolerance for change in chi-squared."""
+
     maxFuncEvals: int = Field(10000, gt=0)
+    """[SIMPLEX] The maximum number of function evaluations before the algorithm terminates."""
+
     maxIterations: int = Field(1000, gt=0)
+    """[SIMPLEX] The maximum number of iterations before the algorithm terminates."""
+
     # Simplex and DE
     updateFreq: int = 1
+    """[SIMPLEX, DE] How often to print out progress to the terminal, in iterations."""
```

Comment:
I think this could be rephrased - something like "Number of iterations between printing progress updates to the terminal"